be/src/exprs/jit/ir_helper.h

```diff
@@ -50,6 +65,14 @@ struct LLVMColumn {
     llvm::Type* value_type = nullptr;  ///< Represents the type of the column's values.
 };
 
+struct JITContext {
+    llvm::Value* index_phi;
+    std::vector<LLVMColumn>& columns;
+    llvm::Module& module;
+    llvm::IRBuilder<>& builder;
+    int input_index;
```

Comment:
Initialization is required here

Reply:
just used once with values. I will init it to 0 in next PR. https://github.com/StarRocks/starrocks/pull/40166/files